components/gitpod-db/src/typeorm/workspace-db-impl.ts

```diff
@@ -430,7 +430,7 @@ export class TypeORMWorkspaceDBImpl extends TransactionalDBImpl<WorkspaceDB> imp
 
     public async findRegularRunningInstances(userId?: string): Promise<WorkspaceInstance[]> {
         const infos = await this.findRunningInstancesWithWorkspaces(undefined, userId);
-        return infos.filter((info) => info.workspace.type === "regular").map((wsinfo) => wsinfo.latestInstance);
+        return infos.map((wsinfo) => wsinfo.latestInstance);
```

Comment:
this method is used in contexts where don't want to include prebuilds. So you might end up not able to start a workspace because there are too many prebuilds running under your user. We should actually go away from treating prebuilds as user-owned. I think we should introduce a new db layer method that finds instances by workspace region.

Reply:
Whoops, I changed the wrong method :grimacing: Thx for checking!